desired_distance: 1.0
alpha: 0.1

